sdk/eventhub/event-hubs/test/receiver.spec.ts

```diff
@@ -61,10 +76,25 @@ describe("EventHub Receiver", function(): void {
 
   describe("with partitionId 0 as number", function(): void {
     it("should not throw an error", async function(): Promise<void> {
-      receiver = client.createConsumer(EventHubConsumerClient.defaultConsumerGroupName, 0 as any, {
-        sequenceNumber: 0
+      let subscription: Subscription | undefined;
+      await new Promise((resolve, reject) => {
+        subscription = consumerClient.subscribe(
+          0 as any,
+          {
+            processEvents: async () => {
+              resolve();
```

Comment:
Is `maxWaitTimeInSeconds` the only reason this test actually resolves? I don't see you sending any test messages.

If so, why is 2 seconds enough? Could it be 0?

Reply:
Not sending any messages as what we want to test is that we dont get any error. Changed the timeout to 0.

Reply:
Sounds good.